### ISSUE TITLE
CMake Error

### ISSUE THREAD

**Issue by ghost** [gammasoft71/Examples_wxWidgets#16]
```
CMake Deprecation Warning at src/Controls/StaticText/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```
`WX version: wxwidgets-gtk3-3.2.2.1-2`
`CMake version: cmake-3.27.1-1`

i can fix it by changing cmake version but for others i left it here

**Comment by gammasoft71** [comment]
Hello,

First of all thank you for your interest to Examples_wxWidgets

Yes, you can make the correction.

I will then include your improvement in the rest of the project.

Thanks in advance.

Regards,
Gammasoft

**Comment by gammasoft71** [comment]
Fixed by @oNddleo #17
